README.md

```diff
@@ -13,22 +13,24 @@ This section guides you in using the Bold BI dashboard in your ASP.NET core samp
 
 The samples require the following requirements to run.
 
- * [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/)
  * [.NET Core 6.0](https://dotnet.microsoft.com/download/dotnet-core)
+ * [Visual Studio Code](https://code.visualstudio.com/download)
 
  ## Using the ASP.NET Core sample
  
- * Open the solution file `BoldBI.Embed.Sample.sln` in Visual studio. 
+ * Open the ASP.NET Core sample in Visual Studio Code.
 
- * Open the EmbedProperties.cs file in the following location, /Models/EmbedProperties.cs.
-
- * Please change the following properties in the `EmbedProperties.cs` file as per your Bold BI Server.
+ * This [link](https://help.boldbi.com/cloud-bi/site-administration/embed-settings/) using able to download the embedConfig.json file. 
```

Comment:
Grammatical mistake in this line. Please correct with available online tool.

Reply:
Online tool using grammatical mistakes resolved.